event_registration_mass_mailing/views/email_template.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" ?>
+<openerp>
+<data>
+
+<record id="email_template_form_minimal" model="ir.ui.view">
+    <field name="name">Email Template - model_id domain</field>
+    <field name="model">email.template</field>
+    <field name="inherit_id" ref="mass_mailing.email_template_form_minimal"/>
+    <field name="arch" type="xml">
+        <field name="model_id" position="attributes">
+            <attribute name="domain">[('model', 'in', ['res.partner', 'mail.mass_mailing.contact', 'event.registration'])]</attribute>
```

Comment:
This makes that another module making the same overwrites your domain. You should inject it in the fields_view_get method instead.